pkg/api/serialization_proto_test.go

```diff
@@ -38,7 +38,7 @@ import (
 
 func init() {
 	codecsToTest = append(codecsToTest, func(version unversioned.GroupVersion, item runtime.Object) (runtime.Codec, error) {
-		s := protobuf.NewSerializer(api.Scheme, runtime.ObjectTyperToTyper(api.Scheme))
+		s := protobuf.NewSerializer(api.Scheme, runtime.ObjectTyperToTyper(api.Scheme), "application/arbitrary.content.type")
```

Comment:
My concern is that it *shouldn't* affect any of the serialization - it should only impact whatever the default value selected is in the nesting.  My worry is that mime type is a very special value that needs to be kept located in very specific places, not littered throughout this code.  Serialization doesn't depend on mime types, only nested encoding does (and nested encoding should probably be via a nested encoder that would specify a specific type).